### PR TITLE
Add device tracker support for Cisco Mobility Express

### PR DESCRIPTION
**Pull request by fbradyirl**
**Description:**
This platform adds device tracker support for Cisco Mobility Express wireless controllers.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant):** home-assistant/home-assistant#21531

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Comment by fbradyirl** [comment]
@frenck apologies I had some issues cleaning the commits up during rebase so I instead re-forked and re-opened this new PR. Thanks.

File: source/_components/device_tracker.cisco_mobility_express.markdown [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+title: "Cisco Mobility Express"
+description: "Instructions on how to integrate Cisco Mobility Express wireless controllers into Home Assistant."
+date: 2019-02-27 11:59
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: cisco.png
+ha_category: Presence Detection
+ha_release: 0.90
```
Comment (MartinHjelmare) [inline source/_components/device_tracker.cisco_mobility_express.markdown]:
```suggestion
ha_release: "0.90"
```
Quotes are needed to keep the trailing zero.

File: source/_components/device_tracker.cisco_mobility_express.markdown [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    password: YOUR_ADMIN_PASSWORD
+```
+
+{% configuration %}
+host:
+  description: The IP address of your controller, e.g., 192.168.10.150.
+  required: true
+  type: string
+username:
+  description: The username of an user with administrative privileges.
```
Comment (MartinHjelmare) [inline source/_components/device_tracker.cisco_mobility_express.markdown]:
```suggestion
  description: The username of a user with administrative privileges.
```

File: source/_components/device_tracker.cisco_mobility_express.markdown [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  required: true
+  type: string
+password:
+  description: The password for your given admin account.
+  required: true
+  type: string
+{% endconfiguration %}
```
Comment (MartinHjelmare) [inline source/_components/device_tracker.cisco_mobility_express.markdown:51]: We should document ssl and verify_ssl options, right?

Comment (fbradyirl) [reply source/_components/device_tracker.cisco_mobility_express.markdown:51]: Yes will do that now.

**Review by MartinHjelmare — APPROVED** [review]

**Comment by haffi78** [comment]
What version is supported here, I tried to integrate both via SSL and without, and I always get cannot setup , auth failed. ( however I know that this is right user and pass )?

I am using 8.8.120.0

Here is the error 
```
Error setting up platform legacy
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/device_tracker/setup.py", line 69, in async_setup_legacy
    self.platform.get_scanner, hass, {DOMAIN: self.config})
  File "/usr/local/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/usr/src/homeassistant/homeassistant/components/cisco_mobility_express/device_tracker.py", line 36, in get_scanner
    config.get(CONF_VERIFY_SSL))
  File "/usr/local/lib/python3.7/site-packages/ciscomobilityexpress/ciscome.py", line 38, in __init__
    self.get_system_info()
  File "/usr/local/lib/python3.7/site-packages/ciscomobilityexpress/ciscome.py", line 43, in get_system_info
    json_data = self._call_api(url)
  File "/usr/local/lib/python3.7/site-packages/ciscomobilityexpress/ciscome.py", line 90, in _call_api
    raise CiscoMELoginError("Failed to authenticate "
ciscomobilityexpress.exceptions.CiscoMELoginError: Failed to authenticate with Cisco Mobility Express controller, check your username and password.
```

**Comment by MartinHjelmare** [comment]
Please open an issue if you suspect a bug. If you need help please use our help channels:
https://home-assistant.io/help/#communication-channels

Merged PRs should not be used for support or bug reports. Thanks!

**Pull request merged**